telltime/Speech Recognition/SpeechRecognitionCore.swift

```diff
@@ -0,0 +1,126 @@
+import SwiftUI
+import ComposableArchitecture
+import Combine
+import Speech
+import SwiftSpeechCombine
+
+struct SpeechRecognitionState: Equatable {
+    var status: SpeechRecognitionStatus = .notStarted
+    var authorizationStatus = SFSpeechRecognizerAuthorizationStatus.notDetermined
```

Comment:
Using explicit type like above is nicer
```suggestion
    var authorizationStatus: SFSpeechRecognizerAuthorizationStatus = .notDetermined
```